core/src/main/java/org/infinispan/statetransfer/StateTransferInterceptor.java

```diff
@@ -416,7 +420,7 @@ private Object handleNonTxWriteReturn(InvocationContext rCtx,
       while (ce instanceof RemoteException) {
          ce = ce.getCause();
       }
-      if (!(ce instanceof OutdatedTopologyException) && !(ce instanceof SuspectException))
+      if (!(ce instanceof OutdatedTopologyException) && !(ce instanceof SuspectException) && !(ce instanceof AllOwnersLostException))
```

Comment:
Yes, e.g. when a backup needs previous value, tries to fetch that from remote nodes and finds out that all are dead...